include/FreeRTOS_TCP_WIN.h

```diff
@@ -229,6 +229,27 @@
                                 uint32_t ulFirst,
                                 uint32_t ulLast );
 
+/**
+ * @brief Check if a > b.
```

Comment:
nit: we might add a little more detail in this description – it could be helpful to know a little more about how `a` and `b` are used.  This helper function is for rolling counter applications like TCP sequence numbers.